tests/php/StoreApi/Routes/ControllerTestCase.php

```diff
@@ -95,9 +95,9 @@ public function assertEqualSets( $expected, $actual ) {
 	/**
 	 * Custom assertion of an API response to confirm data and response code is expected.
 	 *
-	 * @param string $endpoint_or_request Route endpoint to get.
-	 * @param int    $expected_response_code Expected response code.
-	 * @param array  $expected_response_data Expected response data.
+	 * @param mixed $endpoint_or_request Route endpoint to get.
```

Comment:
Should this be `string|\WP_Rest_Request`?

Reply:
Yes it should, fixed!